apps/staking/src/api.ts

```diff
@@ -54,7 +54,8 @@ type Data = {
     poolUtilizationDelta: bigint;
     numFeeds: number;
     qualityRanking: number;
-    apyHistory: { date: Date; apy: number }[];
+    delegationFee: bigint;
+    apyHistory: { date: Date; apy: number; selfApy: number }[];
```

Comment:
Hmm in my opinion it's better to pass the current stake account to the sdk when fetching the apy, rather than having all the raw data and calculating it all in the frontend, thoughts?